packages/react-native/ReactAndroid/src/test/java/com/facebook/react/bridge/BaseJavaModuleTest.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package com.facebook.react.bridge
+
+import com.facebook.react.turbomodule.core.interfaces.TurboModule
+import com.facebook.testutils.shadows.ShadowSoLoader
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when` as whenever
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+/** Tests for [BaseJavaModule] and [JavaModuleWrapper] */
+@Config(shadows = [ShadowSoLoader::class])
+@RunWith(RobolectricTestRunner::class)
+class BaseJavaModuleTest {
+  private lateinit var methods: List<JavaModuleWrapper.MethodDescriptor>
+  private lateinit var moduleWrapper: JavaModuleWrapper
+  private lateinit var generatedMethods: List<JavaModuleWrapper.MethodDescriptor>
+  private lateinit var generatedModuleWrapper: JavaModuleWrapper
+  private lateinit var arguments: ReadableNativeArray
+
+  @Before
+  fun setup() {
+    val moduleHolder = ModuleHolder(MethodsModule())
+    moduleWrapper = JavaModuleWrapper(null, moduleHolder)
+    methods = moduleWrapper.methodDescriptors
+    val generatedModuleHolder = ModuleHolder(GeneratedMethodsModule())
+    generatedModuleWrapper = JavaModuleWrapper(null, generatedModuleHolder)
+    generatedMethods = generatedModuleWrapper.methodDescriptors
+    arguments = mock(ReadableNativeArray::class.java)
+  }
+
+  private fun findMethod(mname: String, methods: List<JavaModuleWrapper.MethodDescriptor>?): Int = methods.indexOfFirst({it.name === mname})
+
```

Comment:
This should be `List<JavaModuleWrapper.MethodDescriptor>` (without `?` at the end).